geocity/apps/submissions/admin.py

```diff
@@ -258,29 +262,39 @@ def get_list_display(self, request):
         return list_display
 
     # Fields used in search_fields and list_filter
-    integrator_fields = [
+    superadmin_search_fields = [
```

Comment:
Superuser not superadmin. Keep correct naming to prevent mistakes like "pilot" and "secrétariat"

Reply:
FMY: what's the difference between "pilot" and "secrétariat"?

Reply:
There is no difference. We should standardize that, also in doc.